quickstarts/observability-as-code/puppet/config.yml

```diff
@@ -0,0 +1,49 @@
+id: b70291ad-ff88-4ff9-9173-d4b2f99656af
+
+# Name of the quickstart (required)
+name: puppet
+
+# Displayed in the UI (required)
+title: Puppet
+
+# Long-form description of the quickstart (required)
+description: |
+  Puppet is a configuration management tool used to manage various stages of the
```

Comment:
More detail here would be helpful for the user to understand the value of this quickstart.